developerportal/apps/people/templates/people.html

```diff
@@ -20,5 +20,8 @@ <h2>People</h2>
         {% include "organisms/newsletter-signup.html" %}
       </section>
     </div>
+    <section id="newsletter" class="section-full section-background">
```

Comment:
Can you revert this change? Looks like a bad merge.

Reply:
Removed